tests/system/python/e2e/test_e2e_modbus_c_rtu_pi.py

```diff
@@ -0,0 +1,178 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Test system/python/e2e/test_e2e_modbus_c_rtu_pi.py
+
+"""
+
+import http.client
+import json
+import time
+import pytest
+import serial
```

Comment:
no module serial.

looks we need to add pyserial as test dep?

Reply:
added